Gems/Atom/RHI/Code/Include/Atom/RHI/ScopeProducer.h

```diff
@@ -89,7 +89,7 @@ namespace AZ
              *  Sets ID of the scope producer. Used by class that inherit from
              *  ScopeProducer but that can't supply a ScopeId at construction.
              */
-            void SetScopeId(const ScopeId& scopeId);
+            void InitScope(const ScopeId& scopeId, HardwareQueueClass hardwareQueueClass = HardwareQueueClass::Graphics);
```

Comment:
You need to go through deprecation process to make such an API change.
https://github.com/o3de/sig-core/blob/main/governance/API-Ref-Guidelines-Update.md#deprecations

Reply:
I also prefer to have two functions for each of them. 

Keep the SetScropeId()
and add a new one for set hardware queue.